docs/agents/mobile.md

```diff
@@ -4,13 +4,56 @@ description: Ship logs from iOS and Android apps to Sematext
 Use the following two libraries for shipping logs from mobile applications and devices running Android and iOS to Sematext Logs.
 Both libraries have the following in common:
 
+- automatic and manual geolocation tagging
 - ship logs in configurable bulk requests to minimize battery and network usage
 - buffer logs while the device is disconnected from the network to avoid data loss
 - additional parameters to control when logs should or should not be shipped
-- meta data with application name and version, OS name and version, and application UUID
+- pause and resume log shipping on demand
+- metadata with application name and version, OS name and version, and application UUID
 - ability to catch and ship uncaught exceptions
 
+Use Sematext Cloud **Mobile Application Logs** Logs App type to get out of the box reports giving you the most important information regarding your mobile applications.
```

Comment:
Logs App type => App
giving you => with
regarding => about

4th grade level words :)

Reply:
Done, thanks :)